wfl/Profiles/RestaurantProfile.cs

```diff
@@ -35,11 +35,28 @@ protected override void Configure(EntityConfiguration<Restaurant> configuration)
                     .WithLink<Restaurant, FoodOrdersController>(c => c.CreateFoodOrderAsync(null))
                     .WithField(x => x
                         .WithName(nameof(FoodOrderDto.RestaurantId))
-                        .WithType("text")
+                        .WithType("hidden")
                         .WithTitle("Restaurant Id")
                         .WithValue(r => r.Id)
                     )
                )
+               .UseActionTransform(actions => actions
+                    .WithName("edit-restaurant")
```

Comment:
I think the purpose of the action is still clear to the user with `edit`. I'll include this change.